### ISSUE TITLE
upload failed when `backup_multipart_chunksize` is much smaller than the size of compressed data in each segment

### ISSUE THREAD

**Issue by weicao92** [greenplum-db/gpbackup-s3-plugin#50]
### TL; DR
It is very easy to see, that when `backup_multipart_chunksize` is much smaller than the size of compressed data in each segment,        lines like
`[DEBUG]:-Https request attempt 0 failed. Next attempt in 49.507618ms. RequestTimeout: Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.`

will flood the `gpbackup-s3-plugin`'s log file, and finally will fail after 10 tries.

### details
/path/to/gpbakcup-s3-plugin-config.yaml
```
executablepath: /path/to/gpbackup_s3_plugin
options:
  region: <region>
  aws_access_key_id: <aws_access_key_id>
  aws_secret_access_key: <aws_secret_access_key>
  bucket: <bucket_name>
  folder: <folder_name>
  encryption: off
  backup_multipart_chunksize: 5MB
```

The target table is called "large_table" (in the database "testdb"), of which each segment has about `2GB `of data after being compressed using gzip.

When we try to backup "large_table" and upload the data to s3 using the following command:
`gpbackup --dbname testdb  --plugin-config /path/to/gpbakcup-s3-plugin-config.yaml --debug --include-table public.large_table `
 
It's easy to observe the behavior articulated in the section "TL;DR", ended up crying out "failed" eventually.

**Comment by oakbarrett** [comment]
@weicao92  backup_multipart_chunksize is a configurable parameter with a default of 250MB.  
Is there a reason you are using 5MB? 

I was unable to reproduce using 5MB and a large table.
